cpp/velox/compute/WholeStageResultIterator.cc

```diff
@@ -301,16 +301,22 @@ void WholeStageResultIterator::collectMetrics() {
     return;
   }
 
+  const auto& taskStats = task_->taskStats();
+  if (taskStats.executionStartTimeMs == 0) {
+    LOG(INFO) << "collectMetrics failed, taskStats is zero, maybe task never call next().";
```

Comment:
`Skip collect task metrics since task did not call next()`

Reply:
Fixed.

Reply:
@ulysses-you Can you review again?